runtime/doc/api.txt

```diff
@@ -1863,7 +1864,7 @@ nvim_parse_cmd({str}, {opts})                               *nvim_parse_cmd()*
           • filter: (dictionary) |:filter|.
             • pattern: (string) Filter pattern. Empty string if there is no
               filter.
-            • force: (boolean) Whether filter is inverted or not.
+            • force: (boolean) Whether the filter is inverted or not.
```

Comment:
This is a good change. The `:help dev-doc` guidance against articles is only for the first part of e.g. parameter descriptions and return-value descriptions where "A ..." and "The ..." tend to add limitless noise to the descriptions.